testing/debug/src/main/scala/stepping/ForComprehensionListObject.scala

```diff
@@ -4,7 +4,7 @@ object ForComprehensionListObject {
 
   def main(args: Array[String]) {
 
-    val l= List(new Object(), "deux", "quatre", "huit")
```

Comment:
aah, for stepping in multi parameter lists. good point. I'm not sure how to do that.